src/main/java/org/broadinstitute/hellbender/bench/BenchMarkDups2.java

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.hellbender.bench;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.common.base.Stopwatch;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.SamReaderFactory;
+import htsjdk.samtools.ValidationStringency;
+import org.broadinstitute.hellbender.tools.dataflow.transforms.markduplicates.MarkDuplicatesFromShardsDataflowTransform;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.SAMRecordToGATKReadAdapter;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * To use the profiler, we need a decent-sized input. Decent-sized input makes Dataflow's direct runner choke.
+ * Hence this little one-off program to exercise file IO, conversion, and MarkDuplicates.
+ */
+public class BenchMarkDups2 {
+
+    public static void main(String[] args) {
```

Comment:
It probably doesn't belong as a full-on CommandLineProgram so perhaps I should remove it. Another option, though, would be to put it in the test tree if you prefer to keep it around.